src/main/app/src/constants/Navbar.js

```diff
@@ -9,7 +9,7 @@ const Navbar = () => {
   const [isLoggedIn, updateIsLoggedIn] = React.useState(false);
 
   // Initialize google auth api information
-  const clientID = "";
+  const clientID = "298755462-7fnqhho2db1b0mb5i6o7irgg8v7om76g.apps.googleusercontent.com";
```

Comment:
We don't want the clientID on github so what I do is I just add it before testing my code and remove it later. Ideally a servlet should provide this.